Modules/ThirdParty/HDF5/src/itkhdf5/CMakeLists.txt

```diff
@@ -39,9 +43,14 @@ endif ()
 
 # CMake version 3.14 added option --ignore-eol to compare files
 #     cmake -E compare_files --ignore-eol file1 file2
-#if(CMAKE_VERSION VERSION_LESS "3.14.0" AND WIN32)
-#    MESSAGE(FATAL_ERROR "Windows builds requires a minimum of CMake 3.14")
-#endif()
+set (CMAKE_IGNORE_EOL "--ignore-eol")
+if (CMAKE_VERSION VERSION_LESS "3.14.0")
+  set (CMAKE_IGNORE_EOL "")
+  if (WIN32)
+    message (FATAL_ERROR "Windows builds requires a minimum of CMake 3.14")
```

Comment:
This also increases CMake version requirement.

Reply:
3.14 was released on 2019-03-14.  Yes, I think I remember that π release!

So, yeah, someone needs to decide about increasing min cmake or not.

Reply:
While I like :pie: :yum:, we should forgo the CMake minimum required bump until ITK 5.3. We have systems running Ubuntu 18.04, and we can resolve the minimum required versions on CI systems, etc. after the 5.2.0 release.

Reply:
I'm a little out of the ITK loop...  Is 5.2 branched? What's master currently? What's release currently?

Reply:
As of this morning `master` and `release` were equivalent -- we are in the release candidate phase for 5.2.